addon-test-support/@ember/test-helpers/dom/click.ts

```diff
@@ -69,6 +71,8 @@ export function __click__(element: Element | Document, options: MouseEventInit):
   click('button', { shiftKey: true });
 */
 export default function click(target: Target, options: MouseEventInit = {}): Promise<void> {
+  const mergedOptions = assign({ buttons: MOUSE_EVENT_BUTTONS.PRIMARY }, options);
```

Comment:
Should we explicitly default `button` also? The tests for it that existed previously passed because `button` is `0` when it is not initialized, but adding it to the `DEFAULT_CLICK_OPTIONS` explicitly seems much easier to reason about to me.

https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button

Reply:
Using the suggestion I made above, this would look like:

```suggestion
  const mergedOptions = assign({}, DEFAULT_CLICK_OPTIONS, _options);
```

Reply:
I had `button` originally (see first commit) but felt it was unnecessary. I will add it back in for clarity.